google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseClientImplTest.java

```diff
@@ -3966,7 +3966,7 @@ public void testCreateSessionsFailure_shouldNotPropagateToCloseMethod() {
     try {
       // Simulate session creation failures on the backend.
       mockSpanner.setCreateSessionExecutionTime(
-          SimulatedExecutionTime.ofStickyException(Status.RESOURCE_EXHAUSTED.asRuntimeException()));
+          SimulatedExecutionTime.ofStickyException(Status.PERMISSION_DENIED.asRuntimeException()));
```

Comment:
This change is not directly relevant for the change in this PR, but I noticed that this test took ~30 seconds to execute. The reason for that is that `RESOURCE_EXHAUSTED` is now a retriable error, which means that it will automatically be retried by Gax for up to 30 seconds before failing.